maintainers/maintainer-list.nix

```
@@ -13699,6 +13699,12 @@
     githubId = 801525;
     name = "rembo10";
   };
+  remexre = {
+    email = "nathan+nixpkgs@remexre.com";
+    github = "remexre";
+    githubId = 4196789;
+    name = "Nathan Ringo";
+  };
```

```suggestion
  remexre = {
    email = "nathan+nixpkgs@remexre.com";
    github = "remexre";
    githubId = 4196789;
    name = "Nathan Ringo";
  };
```
please split this out into it's own commit called: `maintainers add remexre`